src/todo_list.js

```diff
@@ -0,0 +1,17 @@
+
+const $inputSchedule = document.getElementById("new-todo-title");
+
+$inputSchedule.addEventListener('keypress', addSchedule);
+console.log("jkkkk")
+
+function addSchedule(event) {
+    let ENTER_KEY = 13
+    console.log(event.key);
+    if(event.keyCode == ENTER_KEY) {
```

Comment:
```suggestion
    if(event.keyCode === ENTER_KEY) {
```
비교 연산자로 `===`를 사용하는 것과는 어떤 차이가 있을까요?
개발자들마다 `==` 연산자를 쓸지 아니면 `===`연산자를 쓸지에 대해서는 의견이 일부 갈릴 수도 있는데, 차이만 확실히 알고 계시면 좋을 것 같습니다 :)